app/api/files/filesystem.ts

```diff
@@ -47,12 +47,34 @@ async function deleteFiles(files: FilePath[]) {
 const testingUploadPaths = {
   uploadedDocuments: `${__dirname}/specs/uploads/`,
   attachments: `${__dirname}/specs/uploads/`,
-  customUploads: `${__dirname}/specs/customUploads/`,
+  customUploads: `${__dirname}/specs/uploads/`,
   temporalFiles: `${__dirname}/specs/uploads/`,
 };
 
-const setupTestUploadedPaths = () => {
-  testingTenants.changeCurrentTenant(testingUploadPaths);
+const createDirIfNotExists = async (dirPath: string) => {
+  try {
+    await asyncFS.mkdir(dirPath);
+  } catch (e) {
+    if (!e.message.match(/file already exists/)) {
+      throw e;
+    }
+  }
+};
+
+const generateUploadsPath = async (subPath: string) => {
+  if (subPath) {
+    await createDirIfNotExists(`${__dirname}/specs/uploads/${subPath}`);
+  }
+  return {
+    uploadedDocuments: `${__dirname}/specs/uploads/${subPath}`,
+    attachments: `${__dirname}/specs/uploads/${subPath}`,
+    customUploads: `${__dirname}/specs/uploads/${subPath}`,
+    temporalFiles: `${__dirname}/specs/uploads/${subPath}`,
```

Comment:
Same concern as above,! Maybe this solves it if we are using only one at a time in each test?

Reply:
this was not being used anymore actually, it has been removed now.